packages/cli-vector/src/modify/layers/contours.ts

```diff
@@ -0,0 +1,104 @@
+import { LogType } from '@basemaps/shared';
+
+import { VectorCreationOptions } from '../../stac.js';
+import { VectorGeoFeature } from '../../types/VectorGeoFeature.js';
+
+/**
+ * Processes a 'contours' layer feature.
+ *
+ * @param feature - the feature to process
+ * @param options - the layer's options
+ * @param logger
+ * @returns the processed feature
+ */
+export function handleLayerContours(
+  feature: VectorGeoFeature,
+  options: VectorCreationOptions,
+  logger: LogType,
+): VectorGeoFeature {
+  logger.debug({}, 'HandleContours:Start');
```

Comment:
Too much similar logs, I wonder we want to set --verbose to false to disable debug logs as default. Otherwise, we got one log for each feature.